### DISCUSSION TITLE
Pagination's `page` property conflicts with `page` CSS attribute.

### DISCUSSION THREAD

**Discussion by AlphaNecron** [chakra-ui/zag#1009] [Ideas]
# 🚀 Feature request

Pagination's `page` property is overshadowed by CSS attribute.

## 🧱 Problem Statement / Justification

Pagination's `page` property will not work when using with styling libraries as `page` value will be passed to `page` CSS attribute instead of `page` property.

## ✅ Proposed solution or API

Replace `page` property with either `currentPage` or `activePage` to ensure compatibility with other styling libraries.

**Comment by segunadebayo** [comment]
Hi @AlphaNecron,

I'd recommend using a styling solution that avoids this problem or allows you to define a `shouldForwardProp` to make sure `page` is forwarded to the machine and not treated as a style prop.

Chakra UI and Panda CSS allows you do this,

**Reply by AlphaNecron** [reply]
I'm using Panda CSS, how can I do that btw?

**Reply by AlphaNecron** [reply]
Figured out, thanks!

**Reply by AlphaNecron** [reply]
![image](https://github.com/chakra-ui/zag/assets/57827456/b9a12677-e128-46fa-b41f-507a666c187c)
![image](https://github.com/chakra-ui/zag/assets/57827456/63c09f0e-0eed-41d9-a133-2e8daad48630)
Is this the right way to use shouldForwardProp?

**Reply by AlphaNecron** [reply]
Nevermind, recipe was `null`, fixed it. Thanks!
